javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,90 @@
+// Shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function compareLength(arr) {
+  return arr.reduce((x, y) => (x.length <= y.length ? x : y));
```

Comment:
Super clever, well done! We haven't even learned array methods yet, and you're already nailing it!!!

Reply:
We kind of touched them during javascript1 module a bit, so...